_posts/2018-02-02-45-minutes-into-powerapps.md

```diff
@@ -0,0 +1,43 @@
+
+## Hands on
+
+Enough with that already, let’s actually do something. First things first, you need an organisational MS account to unleash the power of PowerApps. After you have one set up head to the [App Store](https://itunes.apple.com/app/id1047318566) or [Play Store](https://play.google.com/store/apps/details?id=com.microsoft.msapps&hl=en) and install PowerApps from Microsoft. Login process needs a bit of patience as clearly we are working in web view environment. Don’t expect to see fingerprint authentication let alone Face ID. After login succeeds and necessary checkboxes are ticked you’ll be watching at the list of your organisations shared apps. Probably not much to see here, yet.
+
+Next, head down to https://web.powerapps.com/home on your favourite browser. On the Home screen you’ll be presented with selection of ordinary business app cases from which to choose, or even start from blank, if you feel adventurous. Going Indiana Jones is way out of this scribble's scope, so we’ll settle with Contacts example. Clicking the “Add” button presents you with permissions dialog from which you need to permit PowerApps to read data from your organisation’s Office 365 Outlook. Obviously it needs to fetch data from somewhere right? I assume Microsoft already has this information covered anyways, so it shouldn’t be an issue to allow access. Also making Contacts app for organisation without actually having organisation’s people on the app is a bit of a let down from end-users’ perspective. Right, so we grant access to app by signing in, again, to our personal account. After that, hit Make this app. And guess what, that is pretty much it. You now have fully functional Contacts app, connected to your organisations Office 365 data, ready to be shared. Template app of course needs to be customised UI wise but search logic, navigation etc. is there 100% ready to run. Nice.
+
+![](/img/powerapps/contacts-app-editor.png)
+
+Fiddling around with styles and nuances of UI is quite similar experience with all the other IDE’s we are accustomed to. Changing colors, fonts, borders, the usual stuff is straightforward, adding images and media needs a bit more imagination. Hit File -> Media -> Images. Then Browse your media files to upload them to service. After upload, you’ll see your files on the list and can give them name, or label if you will. From thereof you can use them as image elements' source by that label. Same goes for video and audio content as well.
```

Comment:
with UI styles and nuances
quite a similar experience
straightforward, but adding images
upload them to the service
After the upload,
give them a name or label.